GerritDownloader/src/main/java/com/holmsted/gerrit/downloaders/ssh/SshDownloader.java

```diff
@@ -287,6 +287,11 @@ private String createProjectNameList() {
         if (getProjectName().isEmpty()) {
             throw new IllegalStateException("No project name defined!");
         }
-        return String.format("project:^%s", getProjectName());
+        String date = getAfterDate();
+        if (date != null) {
+            return String.format("project:^%s after:%s", getProjectName(), date);
```

Comment:
"after:" introduced in Gerrit v2.9. cf. https://gerrit-review.googlesource.com/#/c/54555/
I made a change here to add this parameter to query only if version is higher than 2.9.
Please tell me if this is ok for you.